pkg/kubelet/apis/deviceplugin/v1beta1/api.proto

```diff
@@ -102,6 +106,10 @@ message PreStartContainerResponse {
 // - Allocate allows Device Plugin to run device specific operations on
 //   the Devices requested
 message AllocateRequest {
+	repeated ContainerAllocateRequest requests = 1;
```

Comment:
Can we call these container_requests?
If we later agree on adding init_containers_requests, it would add some consistency